gnuradio-runtime/include/gnuradio/tagged_stream_block.h

```diff
@@ -134,7 +134,7 @@ namespace gr {
      * \param input_items See gr::block
      * \param output_items See gr::block
      */
-    virtual int work(int noutput_items,
+    virtual int work(size_t noutput_items,
```

Comment:
Any changes to "work" or "general_work" like this one have the potential to break a -lot- of OOT modules because the method prototype will be different between the initial declaration here and the one trying to be overloaded in the OOT. Need to be -very- careful here!